app/views/edit_phone/code.html.erb

```diff
@@ -1,3 +1,4 @@
+<% content_for :title, t("mfa.phone.code.change_heading") %>
 <%= render "govuk_publishing_components/components/heading", {
   text: t("mfa.phone.code.change_heading"),
```

Comment:
In this case, we can yield from the title to save including the key again.  I think we should do this in all the views where the heading is the same as the title.
```suggestion
  text: yield(:title),
```